From 48f2c2c46ba1969f3ea8fd2a453ceabb4d9bcc8f Mon Sep 17 00:00:00 2001
From: tracierenea <tracie.perez@mavs.uta.edu>
Date: Tue, 5 Apr 2016 17:59:10 -0500
Subject: [PATCH 03/28] gr-fec: LPDC python function - addressing an
 "undefined" error

Minor fix to address this error:

Traceback (most recent call last):
  File "Generate_LDPC_matrix.py", line 63, in <module>
    [bestH,g] = get_best_matrix(newH,100)
  File "/home/tracie/pybombs/src/gnuradio/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py", line 616, in get_best_matrix
    [betterH, gap, t]
NameError: global name 'betterH' is not defined
---
 gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py b/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py
index c665e9e..6222443 100644
--- a/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py
+++ b/gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py
@@ -613,7 +613,7 @@ def get_best_matrix(H, numIterations=100, verbose=False):
         print 'greedy_upper_triangulation error: ', e
     else:
       if ret:
-        [betterH, gap, t]
+        [betterH, gap, t] = ret
       else:
         continue
 
-- 
2.1.4

